### PR TITLE
Upgrade clean-css package

### LINKED ISSUE #1096: css declaration removed in <style> tag after css minify
Hey team 👋 First thanks for all the work put on this project.

I want to minify a svg that contain a `<style>` tag:

```html
<svg version="1.1" baseProfile="full" width="300" height="86" viewBox="0 0 300 86" preserveAspectRatio="xMinYMin meet" xmlns="http://www.w3.org/2000/svg">
  <style>
    text {
      fill: #EAEAEA;
      font: 500 12px 'Segoe UI', Ubuntu, Sans-Serif;
    }
    .username {
      font-weight: 500;
      font-size: 16px;
    }
    .reputation {
      font-weight: 600;
      font-size: 14px;
    }
    .website, .location {
      font-size: 10px;
    }
    .badgecount circle {
      cx: 0;
      cy: -4;
      r: 4;
    }
    .website {
      fill: #81C4F8;
    }
  </style>

  <rect 
    x="0"
    y="0"
    rx="5"
    ry="5"
    width="300"
    height="86"
    fill="#2D2D2D"
    stroke-width="2"
    stroke="#F48024"
  ></rect>
</svg>
```

But when I minify it on [the website](https://kangax.github.io/html-minifier/)v4.0.0, the css declaration for `.badgecount circle` is removed:

```html
<svg baseProfile=full height=86 preserveAspectRatio="xMinYMin meet" version=1.1 viewBox="0 0 300 86" width=300 xmlns=http://www.w3.org/2000/svg><style>text{fill:#eaeaea;font:500 12px 'Segoe UI',Ubuntu,Sans-Serif}.username{font-weight:500;font-size:16px}.reputation{font-weight:600;font-size:14px}.location,.website{font-size:10px}.website{fill:#81c4f8}</style><rect fill=#2D2D2D height=86 rx=5 ry=5 stroke=#F48024 stroke-width=2 width=300 x=0 y=0></rect></svg>
```

Any idea why it's happening?

### PR DESCRIPTION
**Pull request by johannchopin**
Upgrade clean-css package to version `^5.1.2`. In this version the paramater `urlQuotes` is by default true to keep quoting inside `url()`. For that I needed to update some tests.

Once it's validated I will open a similar PR in  https://github.com/terser/html-minifier-terser.

Closes #1096.

### CODE REVIEW

**Comment by DanielRuf** [comment]
Would be great to have this PR at https://github.com/terser/html-minifier-terser too =)

**Comment by XhmikosR** [comment]
@johannchopin thanks for the PR. Can you please

1. exclude the dist files
2. update to the latest clean-css

Also, weren't there any breaking changes in clean-css v5.x?

**Comment by johannchopin** [comment]
@XhmikosR They was created by the test script. Is there a reason this folder is not already mentioned in the `.gitignore`? 

> exclude the dist files
> update to the latest clean-css

✅ Done

I didn't saw any breaking change acccording to the [changelog](https://github.com/clean-css/clean-css/blob/master/History.md#500--2021-01-29) and the tests are passing so I think we are good.

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "camel-case": "^3.0.0",
-    "clean-css": "^4.2.1",
+    "clean-css": "^5.2.1",
```
Comment (XhmikosR) [inline package.json:58]: Please use npm 6.x.

**Comment by XhmikosR** [comment]
> @XhmikosR They was created by the test script. Is there a reason this folder is not already mentioned in the `.gitignore`?

No idea, but it's probably needed for the website which is hosted from this branch (unfortunately).

> I didn't saw any breaking change acccording to the [changelog](https://github.com/clean-css/clean-css/blob/master/History.md#500--2021-01-29) and the tests are passing so I think we are good.

All right, I'll check this out later and merge it after my last comment is addressed too.

File: package.json (hunk 1 above)
Comment (johannchopin) [reply package.json:58]: Done ✅

**Pull request merged**